packages/catalog-model/src/kinds/util.ts

```diff
@@ -22,9 +22,18 @@ import { KindValidator } from './types';
 // exported kind validators have the `KindValidator` signature which is
 // different. So let's postpone that change until a later time.
 export function ajvCompiledJsonSchemaValidator(schema: unknown): KindValidator {
-  const validator = entityKindSchemaValidator(schema);
+  let cachedValidator = null;
```

Comment:
We avoid `null` unless required ([STYLE.md](https://github.com/backstage/backstage/blob/master/STYLE.md#syntax-and-types))